### ISSUE TITLE
Missing audio volume

### ISSUE THREAD

**Issue by riderx** [l3mpire/lemverse#23]
We cannot chose the volume of our mic and the volume of the people we talk with . ( reported from user of indie verse)

**Comment by Donorhan** [comment]
That's right, I'll have to set up audio worklets for that. I had started to make one to detect the sound of the person speaking and highlight it.
